backend/routes/profiles.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+const { Profile } = require('../models');
+
+router.get("/", async (req, res) => {
+    const response = await Profile.find({});
+    res.json(response);
+  });
+
+  module.exports = router;
```

Comment:
nitpick: remove the indent for this line, also i can show u guys some auto-formatting tools that will make everything formatted when u save the file